onnxscript/function_libs/torch_lib/ops/nn.py

```diff
@@ -114,6 +114,34 @@ def _adjust_attributes_of_avg_pool(
     return (kernel_shape, strides, pads)
 
 
+def _aten_avg_pool_onnx(
+    self: TFloat,
+    kernel_shape: Sequence[int],
+    strides: Sequence[int],
+    pads: Sequence[int],
+    ceil_mode: bool,
+    count_include_pad: bool,
+    unbatched_rank: int,
+) -> TFloat:
+    self_rank_is_unbatched_rank = len(self.shape) == unbatched_rank
```

Comment:
unbatched_rank seems to be len(kernel_shape)+1? Only check it with equal or not is a bit ambiguous.

Reply:
Or add assertion would do too

Reply:
I use `len(kernel_shape) + 1` instead of `unbatched_rank` here. `_aten_max_pool_onnx` could do the same thing.